src/sentry/migrations/0375_groupedmesssage_type_individual_index.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.28 on 2023-03-08 22:04
+
+from django.db import migrations
+
+import sentry.db.models.fields.bounded
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0374_breaking_even_more_keys"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="group",
+            name="type",
+            field=sentry.db.models.fields.bounded.BoundedPositiveIntegerField(
+                db_index=True, default=1
+            ),
```

Comment:
Do you not need the project in this index as well?

Reply:
This index is primarily used for this query:
https://github.com/getsentry/sentry/pull/44645/files#diff-cea78002ba15176d3ee1dbbe210fcc1e15234bd1b30562525d7ee04785a5d213R41-R58

We might end up removing it after the migration is run in SaaS.